### PR TITLE
Hackherthon annual conference

### PR DESCRIPTION
**Pull request by usorfaitheloho**
In this project, I achieved the following;

- used HTML and pure CSS to build the home and about page

- added the speakers dynamically with javascript

- made the pages responsive on both mobile and desktop

- linked the nav-links to the home and about page

- added functionality to the view more button for the speakers.

Attached below is a link to my loom video describing the project
https://www.loom.com/share/fd44adc22eb849078e54bd8a014eec98?sharedAppSource=personal_library

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    <section class="main-body">
+        <div class="container-fluid intro-section body-text">
+            <h3 class=" ml-5 hello">"Are you Ready!!!"</h3>
+            <h1 class="text-effect font-weight-bold p-3 font-style-medium  text-md-left ml-5 creative">HackHerTHon <br/>ANNUAL CONFERENCE<br/> 2022</h1>
+             <p class="ml-5 p-3">A celebration of female professionals who are determined to build digital innovations <br/>it is an extraordinary opportunity for women to showcase their skills in various fields in tech and network with other women in the tech space</p>
+             <h2 class="ml-5 p-3 font-weight-bold mb-3">2022.10.10(SAT) ~ 16(SAT)</h2>
+             <p class="ml-5 p-3 font-weight-bold mb-3">@ ICT Hub Yaba,lagos</p>
+        </div>
+    </section>
+    <section class="container-fluid py-5 main-section d-flex d-flex justify-content-center flex-column align-item-center">
```
Comment (Deepakdanger) [inline index.html:114]: - [ ] Please fix the background image as requirement.
- Your website
sdvsdv
- Original website
sadas

File: index.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+              SEE THE WHOLE PROGRAM
+            </a>
+            <hr class="section_desktop_hr">
+          </div>
+          <hr/>
+        </div>
+    </section>
+  
+
+    <section class="featured_speakers container-fluid">
```
Comment (Deepakdanger) [inline index.html:191]: - [ ] Kindly use the same design & background as used in the original website. Use the same color for showing the designation of the speaker as used in the original website.
- Your design
vdvfdb
- Original website 
fvsdv

File: index.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+              </li>
+              <li class="pr-3">
+                <img src="assets/images/Udemy.png" class="img-partner" alt="Udemy">
+              </li>
+              <li class="pr-3">
+                <img src="assets/images/skoolMedia.png" class="img-partner" alt="skoolMedia">
+              </li>
+              <li > 
+                <img src="assets/images/IBEForuM_logo.png" class="img-partner" width="100" alt="Intel Partner">
+              </li>
```
Comment (Deepakdanger) [inline index.html]: - [ ] Please change the partner section as in original websites. 
- Your website
vsdv
- Original website 
scdcs

File: about.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+            </div>
+            <div class="col-12 col-md-5 ml-md-2 mt-4 text-center text-white p-5 second_img">
+              <h3 class="font-weight-bold mb-3">2020</h3>
+              <p class="font-weight-bold text-white">CC Global Summit 2013 in Warsaw</p>
+            </div>
+          </div>
+      </div>
+</section>
+
+    <div class="container-fluid row col-12 pl-0 pr-0 ml-0 mr-0 mt-3 partners black">
```
Comment (Deepakdanger) [inline about.html]: - [ ] In **Mobile view** of About page , There should be `partner section` before footer , Kindly add `partner section`.
- [ ] In **Mobile view** the background image of the footer should be light. kindly fix it.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @usorfaitheloho, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
## To Highlight
- Good PR ✔️ 
- Website is responsive ✔️ 
- No Linter error ✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
```
Comment (brytebee) [inline about.html:14]: - [ ] **Optional:** Kindly fix the console errors shown below.

image

File: assets/js/script.js [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  </div>
+  <div class="col-9 speaker-block pr-0 ">
+    <h3 class=" section-title font-weight-bold">${data.Name}</h3>
+    <p class="orange font-italic">${data.Position}</p>
+    <span class="hr-speakers ml-0 mb-3"> </span>
+    <p>${data.Occupation}</p>
+  </div>
+  </div>
+</div>
+  `).join('');
```
Comment (brytebee) [inline assets/js/script.js:86]: - [ ] **Optional:** Kindly fix the console errors shown below.

image

**Review by brytebee — APPROVED** [review]
Hi @usorfaitheloho & @Deepakdanger👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (usorfaitheloho) [reply index.html:114]: Thanks for the correction @Deepakdanger .I will adjust the background image to meet the requirement.

File: index.html (hunk 2 above)
Comment (usorfaitheloho) [reply index.html:191]: Thanks for the corrections,I have implemented the changes

File: index.html (hunk 3 above)
Comment (usorfaitheloho) [reply index.html]: Thanks for the correction, it was a mistake on my end. I have implemented the changes

File: about.html (hunk 4 above)
Comment (usorfaitheloho) [reply about.html]: Thanks for the correction,I have added the partner section to the mobile view and implement the changes in the background image

File: about.html (hunk 5 above)
Comment (usorfaitheloho) [reply about.html:14]: thanks for noticing the console errors 😄 , I have fixed it.

**Comment by usorfaitheloho** [comment]
Thanks @brytebee for reviewing and approving my project, I appreciate it 😄

**Pull request merged**